skabase/SKALogger/test/SKALogger_test.py

```diff
@@ -125,8 +124,7 @@ def test_SetElementLoggingLevel(self, tango_context, setup_log_test_device,
     @pytest.mark.parametrize("logging_level", [int(tango.LogLevel.LOG_WARN)])
     @pytest.mark.parametrize("logging_target", ["logger/test/1"])
     # PROTECTED REGION END #    //  SKALogger.test_SetStorageLoggingLevel_decorators
-    def test_SetStorageLoggingLevel(self, tango_context, setup_log_test_device,
```

Comment:
Same comment about fixture being removed.

Reply:
We are instantiating separate container for SKATestDevice. However, for testing in the local environment, we still need this fixture. We had removed this in anticipation that we will do not more testing in the local environment. But as a second thought, we still may need to run test cases locally. So we will revert this change for some time and if not found useful, will remove it.